ui/desktop/app/templates/scopes/scope/authenticate/method/oidc.hbs

```diff
@@ -11,7 +11,7 @@
     <p>{{t 'resources.auth-method.messages.pending.description'}}</p>
     <p>
       {{t 'resources.auth-method.questions.no-see-window'}}
-      <a href={{this.authURL}} target='_blank' rel='noreferrer noopener'>
+      <a href='#' role='button' {{on 'click' this.retryAuthentication}}>
```

Comment:
How come we have the role set to a `button`? Is it not considered a link?

Reply:
ah okay thats a mistake, i thought it was needed for the 'click' handler but its not.